pkg/actions/nodegroup/nodegroup.go

```diff
@@ -18,6 +18,8 @@ type Manager struct {
 	cfg          *api.ClusterConfig
 	clientSet    *kubernetes.Clientset
 	wait         WaitFunc
+	init         eks.NodeGroupInitialiser
```

Comment:
I feel like `init` is a bit vague, but not sure on a better name :man_shrugging:

Reply:
i think `initialiser`, while verbose, would be clearer. `init` as we know it in Go _does_ shorthand initialise, but it may be helpful to clarify that we don't mean _that kind_ of `init`.